creator-node/src/services/stateMachineManager/monitoring/StateMonitoringQueue.js

```diff
@@ -92,42 +106,41 @@ class StateMonitoringQueue {
       this.logError(`Queue Job Error - ${error}. Queuing another job...`)
     })
 
-    // Re-queue a new job when the current job fails or succeeds
+    // Add handlers for when a job fails to complete (or completes with an error) or successfully completes
     queue.on('completed', (job, result) => {
       this.log(
         `Queue Job Completed - ID ${job?.id} - Result ${JSON.stringify(
           result
         )}. Queuing another job...`
       )
       if (result?.jobFailed) {
-        this.enqueueJobAfterSuccess(queue, job, result)
+        jobSuccessCallback(queue, job, result)
       } else {
-        this.enqueueJobAfterFailure(queue, job)
+        jobFailureCallback(queue, job)
```

Comment:
i think these should be flipped? i think you're calling successCallback on failure